cg/models/downsample/downsample_data.py

```diff
@@ -42,7 +42,7 @@ def downsampled_sample_name(
         self,
     ) -> str:
         """Return a new sample name with the number of reads to which it is down sampled in millions appended."""
-        return f"{self.sample_id}_{self.number_of_reads}M"
+        return f"{self.sample_id}-{self.number_of_reads}M"
```

Comment:
Hooold, don't merge yet. "-" might be an issue too! @ivadym is looking into it 🌟